sqlx-core/src/logger.rs

```diff
@@ -98,26 +98,19 @@ impl<'q> QueryLogger<'q> {
             let log_is_enabled = log::log_enabled!(target: "sqlx::query", log_level)
                 || private_tracing_dynamic_enabled!(target: "sqlx::query", tracing_level);
             if log_is_enabled {
-                let mut summary = parse_query_summary(&self.sql);
-
-                let sql = if summary != self.sql {
-                    summary.push_str(" …");
-                    format!(
-                        "\n\n{}\n",
-                        sqlformat::format(
-                            &self.sql,
-                            &sqlformat::QueryParams::None,
-                            sqlformat::FormatOptions::default()
-                        )
+                let sql = format!(
+                    "{}\n",
+                    sqlformat::format(
+                        &self.sql,
+                        &sqlformat::QueryParams::None,
+                        sqlformat::FormatOptions::default()
                     )
-                } else {
-                    String::new()
-                };
+                );
 
                 private_tracing_dynamic_event!(
                     target: "sqlx::query",
                     tracing_level,
-                    summary,
+                    summary = format!("query went over configured duration of {}ms", self.settings.slow_statements_duration.as_millis()),
```

Comment:
As written, the message will be the same even if the query didn't exceed the threshold.

We should also not be changing the contents of these fields.